exonum/src/blockchain/schema.rs

```diff
@@ -331,6 +338,15 @@ where
     fn next_height(&self) -> Height {
         Height(self.block_hashes_by_height().len())
     }
+
+    /// Count txs in pool
```

Comment:
"Counts transactions in the pool" or "returns number of transactions in the pool".